client/components/folder-items.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+export default class FolderItems extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = ({
+      folders: []
+    });
+    this.openFolder = this.openFolder.bind(this);
+  }
+
+  openFolder(folderId) {
+    window.location.href = `http://localhost:3000/#home-page?folderId=${folderId}`;
+  }
```

Comment:
Instead of having a method like this, you should wrap your folders in `<a href=???>`, and let the browser do its thing with the hash changes. If necessary you can set their `display:` to `block`